cli/src/commands/migration/complete.test.ts

```diff
@@ -0,0 +1,199 @@
+                  default: '0',
+                  nullable: false,
+                  unique: false,
+                  comment: ''
+                }
+              },
+              indexes: {},
+              primaryKey: ['xata_id'],
+              foreignKeys: null,
+              checkConstraints: null,
+              uniqueConstraints: null
+            }
+          }
+        }
+      })
+    };
+  }
+
+  throw new Error(`Unexpected fetch request: ${url} ${request.method}`);
+};
+
+export const fetchEmptyStatus = (url: string, request: any) => {
+  if (url === `${baseUrl}/migrations/status` && request.method === 'GET') {
+    return {
+      ok: true,
+      status: 204,
+      json: async () => null // 204, no content
+    };
+  } else {
+    return baseFetch(url, request);
+  }
+};
+
+export const fetchRunningMigrationWithSuccessfulComplete = (url: string, request: any) => {
+  if (url === `${baseUrl}/migrations/status` && request.method === 'GET') {
+    return {
+      ok: true,
+      json: async () => {
+        return {
+          completedAt: '2024-07-22T16:52:55.125163Z',
+          description: [
+            {
+              update: {
+                mapping: {
+                  description: '_pgroll_new_description'
+                },
+                name: 'description',
+                table: 'table',
+                type: 'column'
+              }
+            }
+          ],
+          jobID: 'mig_job_cqf8spc40ertajao726g',
+          status: 'completed',
+          type: 'start'
+        };
+      }
+    };
+  }
+
+  if (url === `${baseUrl}/migrations/complete` && request.method === 'POST') {
+    return {
+      ok: true,
+      json: async () => {
+        return { jobID: 'mig_job_cqfltm0knc8jbigtjb9g' };
+      }
+    };
+  }
+
+  return baseFetch(url, request);
+};
+
+promptsMock.mockReturnValue({ confirm: true, database: 'db1', workspace: 'test-1234' });
+
+describe('migration complete', () => {
+  test('correctly detects if there is no migration to complete', async () => {
+    const config = await Config.load();
+    const command = new MigrationComplete(['main'], config);
+    const error = vi.spyOn(command, 'error');
+    fetchMock.mockImplementation(fetchEmptyStatus);
+    try {
+      await command.run();
+    } catch (e) {
+      console.error(e);
+    }
+    expect(error).toHaveBeenCalledWith(`No active migration found, there is nothing to complete.`);
+  });
+
+  test('correctly starts the migration complete job, if an active migration is found', async () => {
+    const config = await Config.load();
+    const command = new MigrationComplete(['main'], config);
+    const log = vi.spyOn(command, 'log');
+    fetchMock.mockImplementation(fetchRunningMigrationWithSuccessfulComplete);
+    await command.run();
+    expect(log).toHaveBeenCalledWith(
+      `Migration complete started with Job ID mig_job_cqfltm0knc8jbigtjb9g. Please use the xata migration status main command to check its status`
```

Comment:
Should we add quotes around the command: `xata migrate status main` in the log?

Reply:
Good idea, it was already using `chalk.gray` but quotes will make it clearer (and the tests more readable too). Done.